qiskit/providers/aer/noise/errors/quantum_error.py

```diff
@@ -292,367 +400,69 @@ def to_dict(self):
         error = {
             "type": "qerror",
             "operations": [],
-            "instructions": list(self._noise_circuits),
-            "probabilities": list(self._noise_probabilities)
+            "instructions": [self._qc_to_json(qc) for qc in self.circuits],
+            "probabilities": list(self.probabilities)
         }
         return error
 
-    def compose(self, other, front=False):
-        """Return the composition error channel other * self.
-
-        Note that for `front=True` this is equivalent to the
-        :meth:`QuantumError.dot` method.
-
-        Args:
-            other (QuantumError): a quantum error channel.
-            front (bool): If True return the reverse order composation
-                          self * other instead [default: False].
-
-        Returns:
-            QuantumError: The composition error channel.
-
-        Raises:
-            NoiseError: if other cannot be converted into a QuantumError,
-            or has incompatible dimensions.
-        """
-        if front:
-            return self._matmul(other, left_multiply=False)
-        return self._matmul(other, left_multiply=True)
-
-    def dot(self, other):
-        """Return the composition error channel self * other.
-
-        Args:
-            other (QuantumError): a quantum error channel.
-
-        Returns:
-            QuantumError: The composition error channel.
-
-        Raises:
-            NoiseError: if other cannot be converted into a QuantumError,
-            or has incompatible dimensions.
-        """
-        return self._matmul(other, left_multiply=False)
-
-    def power(self, n):
-        """Return the compose of a error channel with itself n times.
-
-        Args:
-            n (int): the number of times to compose with self (n>0).
-
-        Returns:
-            QuantumError: the n-times composition error channel.
-
-        Raises:
-            NoiseError: if the power is not a positive integer.
-        """
-        if not isinstance(n, int) or n < 1:
-            raise NoiseError("Can only power with positive integer powers.")
-        ret = self.copy()
-        for _ in range(1, n):
-            ret = ret.compose(self)
+    @staticmethod
+    def _qc_to_json(qc: QuantumCircuit):
+        bit_indices = {bit: index for index, bit in enumerate(qc.qubits)}
+        ret = []
+        for inst, qargs, _ in qc:
+            name = inst.label if isinstance(inst, UnitaryGate) and inst.label else inst.name
```

Comment:
Umm, I forgot why I did so. It might be a hack to preserve some original behavior, but even if I replace this line with `name=inst.name`, it does not causes any test failures. I'll check tutorials and if no problem, I'll update this line.

Reply:
Updated in 36b40e0